t/tap.c

```diff
@@ -87,8 +86,7 @@ int is_str(const char* have, const char* want, const char *message, ...) {
     do_test( test, message, args );
 
     if( !test ) {
-        diag("have: %s", have);
-        diag("want: %s", want);
+      diag("have: %s, want: %s", have, want);
```

Comment:
have and want are supposed to be on two lines, the two values are lined up to more easily see the difference.